pkg/apiclient/client_factory.go

```diff
@@ -154,7 +163,7 @@ func (c *Client) GetSpacedClient() (*octopusApiClient.Client, error) {
 	var foundSpaceID string
 	// if c.Ask is nil it means we're in automation mode.
 	if c.SpaceNameOrID == "" {
-		if c.Ask == nil {
+		if !c.Ask.IsPromptEnabled() {
```

Comment:
Can we change `IsPromptEnabled` to `IsInteractive` or something like that? I'd like to aim for a consistent term that we use to describe the situation where a human is using our CLI. We've called this "interactive mode" in our docs.

Reply:
sounds good